docs/DEV.md

```diff
@@ -361,3 +361,26 @@ The default Basic Auth username is `admin`, which is written to `/var/lib/faasd/
 * `faasd install` - install faasd and containerd with systemd, this must be run from `$GOPATH/src/github.com/openfaas/faasd`
 * `journalctl -u faasd -f` - faasd service logs
 * `journalctl -u faasd-provider -f` - faasd-provider service logs
+
+#### Uninstall
+
+* Stop faasd and faasd-provider
+```
+sudo systemctl stop faasd
+sudo systemctl stop faasd-provider
+sudo systemctl stop containerd
+```
+
+* Remove faasd local binary from machine
+```
+sudo rm /usr/local/bin/faasd
+sudo rm /usr/lib/systemd/system/faasd-provider.service
+sudo rm /usr/lib/systemd/system/faasd.service
+sudo systemctl daemon-reload
```

Comment:
Remove the `/var/lib/faasd` folder also.

Reply:
done